backend/packages/Upgrade/src/api/services/UserService.ts

```diff
@@ -103,4 +125,48 @@ export class UserService {
     const searchStringConcatenated = `concat_ws(' ', ${stringConcat})`;
     return searchStringConcatenated;
   }
+  
+  public sendWelcomeEmail(email: string): void {
+    const emailSubject = `Welcome to UpGrade!`;
+    const emailBody = `Greetings!, 
+    <br>
+    A new user account was created for you in UpGrade. You can sign into UpGrade using your Google credentials.
+    <br>
+    To know more about how UpGrade works, please visit 
+    <a href="https://www.upgradeplatform.org/"> www.upgradeplatform.org</a>
+    . To read the documentation, visit 
+    <a href="https://upgrade-platform.gitbook.io/upgrade-documentation/"> UpGrade-docs</a>
+    <br>`;
+
+    this.sendEmail(email, emailSubject, emailBody);
+  }
+
+  public sendRoleChangedEmail(email: string, role: UserRole): void {
+    const emailSubject = `UpGrade Role Changed`;
+    const emailBody = `Greetings!, 
+    <br>
+    Your Role in Upgrade is changed to ${role}!
+    <br>
+    To know more about how UpGrade works, please visit 
+    <a href="https://www.upgradeplatform.org/"> www.upgradeplatform.org</a>
+    . To read the documentation, visit 
+    <a href="https://upgrade-platform.gitbook.io/upgrade-documentation/"> UpGrade-docs</a>
+    <br>`;
```

Comment:
Move email template to a separate file for code maintenance.

Reply:
okay